.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes and elaborate on motivation and context. -->
+<!--- Make sure to refer to relevant GitHub issues using # -->
+
+## Tests
+<!--- Put an `x` in all the boxes that apply: -->
+- [ ] No new tests required.
+- [ ] New tests for the added feature are part of this PR.
+- [ ] I validated the changes manually.
+

```

Comment:
Should we have a section asking authors to specify if the change is only python SDK or UX or both?

Reply:
I was thinking that's obvious to anyone looking at the PR. On second thought, this could turn out to be useful if we ever use the descriptions to generate release summaries (without any manual effort). Perhaps something like

npm packages changed:
- responsibleai/fairness
- ...

python packages changed::
- raiwidgets
- responsibleai
- erroranalysis
- ...

Wdyt? Also adding @imatiach-msft @xuke444 who may have thoughts.

Reply:
That sounds good to me.

Reply:
I updated the PR with a list. Note that I've not included other packages we maintain since they aren't in this repo. I may still have missed some, of course.